deploy_config_generator/output/marathon.py

```diff
@@ -374,6 +392,25 @@ def build_secrets(self, app_vars, data):
             if secrets:
                 data['secrets'] = secrets
 
+    def build_networks(self, app_vars, data):
+        networks = []
+        # The 'networks' param overrides the 'docker_network' param, so look for it first
+        if app_vars['APP']['networks']:
+            for net in app_vars['APP']['networks']:
+                networks.append(net)
+        else:
+            # Translate 'docker_network' value to new-style network
+            tmp_network = {}
+            if app_vars['APP']['docker_network'].lower() == 'bridge':
+                tmp_network['mode'] = 'container/bridge'
+            elif app_vars['APP']['docker_network'].lower() == 'host':
+                tmp_network['mode'] = 'host'
+            else:
+                # TODO: do something meaningful for unknown network mode
+                pass
```

Comment:
Maybe just fail, instead? Or simply return whatever was given.

Reply:
A value that's valid for `container.docker.network` isn't valid for `networks.mode` (which is why there's a translation), and adding a failure here can open a bigger can of worms (specifying valid values for parameters).